resources/data/elephpants.json

```
@@ -591,6 +591,14 @@
             "sponsor": "Laravel Community",
             "year": 2023,
             "image": "74-aubrey.jpg"
+        },
+        {
+            "id": 75,
+            "name": "Dyflexis",
```

```suggestion
            "name": "Flexy",
```
As per @dseguy's [comment](https://github.com/jgrossi/elephpant.me/pull/191#discussion_r1412424793).